vibezz/src/app/components/Navbar.tsx

```diff
@@ -0,0 +1,13 @@
+'use client'
+
+const Navbar: React.FC = () => {
+    return (
+        <div>here comes the nav</div>
+    )
+} 
+
+export default Navbar;
+
+
+
+
```

Comment:
Remove the extra line, only one extra line is enough at the end of files

Reply:
ok